calculatorUICOOP/ViewModels/MainPageViewModel.cs

```diff
@@ -106,28 +118,54 @@ public void DeleteLastInput()
             {
                 ClearScreen();
             }
-           
+
         }
 
-        public void ShowRemainderOnDisplay(string remainder)
+        public void Percentage(string percent)
         {
-
+            AssignOperator(percent);
         }
 
         public void ShowDecimalOnDisplay(string decimalDot)
         {
-
+            if (!_hasDecimal)
+            {
+                DisplayContent += decimalDot;
+                _hasDecimal = true;
+            }
         }
 
         public void Equals()
         {
-
+            double _number2 = Convert.ToDouble(DisplayContent);
+            switch (_operator)
+            {
+                case "+":
+                    DisplayContent = MathLogic.Add(_number1, _number2).ToString();
+                    break;
+                case "-":
+                    DisplayContent = MathLogic.Subtract(_number1, _number2).ToString();
+                    break;
+                case "*":
+                    DisplayContent = MathLogic.Multiply(_number1, _number2).ToString();
+                    break;
+                case "/":
+                    if (DisplayContent != "0")
+                        DisplayContent = MathLogic.Divide(_number1, _number2).ToString();
+                    else
+                        DisplayContent = "Can't Divide by 0";
+                    break;
+                case "%":
+                    var tmp = MathLogic.Multiply(_number1, _number2 > 0 ? _number2 : 1);
+                    DisplayContent = MathLogic.ConvertToPercent(tmp).ToString();
```

Comment:
Although this works when you press Equals. The expectation is that pressing the "%" key updates the inputted number to it's percentage representation

Example:
Input 5
Press "%"
Displayed text: 0.05
Without pressing "="
![uEYAJOc38K](https://user-images.githubusercontent.com/58752614/99914725-558aa700-2ccd-11eb-896d-89ca521f8d50.gif)

Reply:
I see. I am still having an issue with the UI project not loading, so I didn't see this. I'll resolve both sometime this afternoon.

Reply:
> Although this works when you press Equals. The expectation is that pressing the "%" key updates the inputted number to it's percentage representation
> 
> Example:
> Input 5
> Press "%"
> Displayed text: 0.05
> Without pressing "="
> ![uEYAJOc38K](https://user-images.githubusercontent.com/58752614/99914725-558aa700-2ccd-11eb-896d-89ca521f8d50.gif)
> 

I made this on a previous commit but, after see my phone's calculator behavior, I decided to change. Please, see my commit:
ff9566e8c5ec99424c65af236b9fc05e11f309d5

Reply:
My phone's calculator gets the left side of the percentage operator times the right side (when it is not zero) and then converts it to a percentage.

Reply:
This is why we have user stories to define the expected/desired behavior of our application (#20). That was we create logic that implements the User Story, and the team knows what the expected behavior should be during testing.

Reply:
@nickmartin1ee7 Ok, sorry for that. But, is there a way to revert this branch to the [penultimate commit](ff9566e8c5ec99424c65af236b9fc05e11f309d5)